docs/source/usage/deployment.rst

```diff
@@ -0,0 +1,118 @@
+
+       “/<Project>/<service>/<stage>/auth_token”
+
+       **Example:** For our Zoom example, this would be something like:
+       
+    .. image:: ../images/Parameter_Store_Example.png
+       :width: 400
+       :alt: /MozDef-EF/zoom/prod/auth_token
+
+  5. In the description field, add “Authentication token used by the Webhook app to post events”.
+  6. For type, select “SecureString”. 
+  7. For value, paste the value from the app’s configuration from your webhook's configured authentication token.
+  8. Add a Tag with key: Project and value: MozDef-EF-<environment>, then create the parameter.
+
+*************************************
+Fill in the Framework Config Template
+*************************************
+
+The following should be done in your local copy of the framework you cloned or forked:
+
+
+  1. Open the downloaded repository in an IDE to edit locally.
+  2. Under config directory, edit “buildspec-prod.yml” file to contain:
+
+    .. code-block:: yaml
+    
+       version: 0.2
+       
+       env:
+         variables:
+           STAGE: prod
+           SERVICE: zoom
+           PROJECT: MozDef-EF
+           API_PATH: zoom
+       
+       phases:
+         install:
+           runtime-versions:
+             python: 3.7
+             nodejs: 10
+           commands:
+             # Install dependencies here
+             - pip3 install --upgrade awscli -q
+             - pip3 install --upgrade pytest -q
+             - pip3 install --upgrade moto -q
+             - pip3 install --upgrade aws-xray-sdk -q
+             - npm install -g --silent --progress=false serverless
+             - npm install --silent --save-dev serverless-pseudo-parameters
+             - npm install --silent --save-dev serverless-prune-plugin
+         pre_build:
+           commands:
+             # Perform pre-build actions here
+             - chmod +x $CODEBUILD_SRC_DIR/config/deploy.sh
+             - $CODEBUILD_SRC_DIR/config/deploy.sh unit-test
+         build:
+           commands:
+             # Invoke the deploy script here
+             - $CODEBUILD_SRC_DIR/config/deploy.sh deploy $STAGE $AWS_REGION
+
+
+  3. The important part here is the filling in of the “env” section on top. These environment variables will be used by the “serverless.yml” file when deployed by the serverless framework.
+     The idea is, for each different source (such as zoom), the service name and API path will be different.
+  4. Save the file.
+  5. Make any other desired changes on the local copy. For the zoom service, there should not be any other changes.
+
+*********************
+Deploy Your Framework
+*********************
+
+This is where we take everything we've done up to this point and start the deployment.
+
+  1. Go back to AWS Console  “Services -> CodeCommit” and create a repository with the name “<project>-<service>”, in this case “mozdef-ef-zoom”. Add a description and a tag with key: Project and value: MozDef-EF-prod.
```

Comment:
We should change the end of this sentence to `"... MozDef-EF-<environment>."`

Reply:
or <project>-<environement>

Reply:
review changes :)

Reply:
Looks good.